docs/reference/search/search-your-data/near-real-time.asciidoc

```diff
@@ -2,7 +2,7 @@
 
 [[near-real-time]]
 === Near real-time search
-The overview of <<documents-indices,documents and indices>> indicates that when a document is stored in {es}, it is indexed and fully searchable in _near real-time_--within 1 second. What defines near real-time search?
+When a document is stored in {es}, it is indexed and fully searchable in _near real-time_--within 1 second. What defines near real-time search?
```

Comment:
I'm now realizing we often have a hyphen in the noun form (there's one here, `_near real-time_`), and arguably we'd need to add one to "near-real-time search" if we're being picky, which isn't worth it. 

So it'd be fair to undo that other change I suggested, and just standardize on the hyphenated form for simplicity and readability.